<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="spirals-vortex.lille.inria.fr" name="org.molgenis.data.support.DynamicEntityTest" tests="5" failures="0" timestamp="25 May 2018 14:03:09 GMT" time="0.003" errors="0">
  <testcase name="setException" time="0.001" classname="org.molgenis.data.support.DynamicEntityTest"/>
  <testcase name="setException" time="0.001" classname="org.molgenis.data.support.DynamicEntityTest"/>
  <testcase name="setNoException" time="0.000" classname="org.molgenis.data.support.DynamicEntityTest"/>
  <testcase name="setException" time="0.000" classname="org.molgenis.data.support.DynamicEntityTest"/>
  <testcase name="setNoException" time="0.001" classname="org.molgenis.data.support.DynamicEntityTest"/>
</testsuite> <!-- org.molgenis.data.support.DynamicEntityTest -->
